scripts/codeshift/newMutation.ts

```diff
@@ -188,14 +147,93 @@ const addToClientSubscription = (camelMutationName: string, subscription?: Lower
   const nextStr = subStr.replace(typename, typename + '\n' + fragment)
   fs.writeFileSync(subPath, nextStr)
 }
+
+const addDummyPermission = (camelMutationName: string) => {
+  const permissionPath = path.join(
+    PROJECT_ROOT,
+    'packages',
+    'server',
+    'graphql',
+    'public',
+    'permissions.ts'
+  )
+  const permissionText = fs.readFileSync(permissionPath, 'utf-8')
+  const root = j(permissionText, {parser: tsParser()})
+  const mutationObject = root.find(j.ObjectProperty, {key: {name: 'Mutation'}}).get()
+  const {properties, start, end} = mutationObject.node.value
+  properties.push(
+    j.objectProperty.from({
+      key: j.identifier(camelMutationName),
+      value: j.identifier('isFIXME')
```

Comment:
+1 Thoughts on adding a comment about what should be changed?  I.e. most new mutations can be safely removed here, since they should usually be allowed via the wildcard.

Reply:
my thinking there is 99% of new mutations will need more than `isAuthenticated`. most will need `isTeamMember` or `isMeetingMember` or something similar. i just want to get a file diff written so the dev doesn't forget to add permissions, not sure how best to do that...